src/main/java/percy/ui/Ui.java

```diff
@@ -0,0 +1,179 @@
+package percy.ui;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import percy.task.Task;
+import percy.task.TaskList;
+
+public class Ui {
+    private static Scanner in;
+    private static PrintStream out;
+
+    /**
+     * Message templates used.
+     */
+    private static final String INDENT = "    ";
```

Comment:
I might have missed it, but there doesn't seem to be use of the constant INDENT in subsequent parts of your code. Do correct me if I am mistaken.